experiments/list_unsupported.sh

```diff
@@ -9,9 +9,7 @@ if [ "$#" -ne 1 ]; then
    exit 2
 fi
 
-path="/home/diffblue/petr.bauch/code/gnat2goto/experiments/$1"
-gtk_path="/home/diffblue/petr.bauch/code/gtkada/src/"
-unit_path="/home/diffblue/petr.bauch/code/gnat2goto/experiments/sparkunit/"
+path=`pwd`/"$1"
```

Comment:
I don't really like that this starts from `pwd`. If we're passed a relative path then that should already be usable as is, and if it's an absolute path this doesn't work